ts/RootContainer.tsx

```diff
@@ -111,6 +112,15 @@ class RootContainer extends React.PureComponent<Props> {
     }
   }
 
+  /**
+   * exitApp will be called if user press twice back when
+   * navigation history is empty. Remember BackHandler.exitApp() works
+   * on Android platforms (and others) but not on iOS platforms
+   */
+  private exitApp = () => {
```

Comment:
you can avoid the curly braces when you have a single statement

```suggestion
  private exitApp = () => BackHandler.exitApp()
```